Executable/execute_injector.py

```diff
@@ -1,28 +1,28 @@
 import sys
-from injector import inject_dll_to_process
-from injector import remote_call_void_func
-from pywinauto import Desktop
+import win32con
+from injector import *
```

Comment:
PEP8: please avoid `import *`. Every function call must be explicit so we can know which module contains this function without scrolling up.